charts/contract-management/templates/configmap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "contract.fullname" . }}
+  labels:
+    {{ include "contract.labels" . | nindent 4 }}
+data:
+  application.yaml: |
+    micronaut:
```

Comment:
I am unhappy with this approach as I would actually need more "additionalProperties" Placeholders to make the whole config changeable. On the other side I don't want to just include the whole config into the values file, as it increases the complexity unnecessarily.
@wistefan How would you approach this?

Reply:
As discussed separately: Leaving in ConfigMap for now, but once we remove the static TIL Config, the configuration should switch to ENV